tutorials/hoon/arms-and-cores.md

```diff
@@ -490,8 +490,23 @@ Again, being able to read Nock is not essential to understanding Hoon.
 
 ### Core Nesting
```

Comment:
This title is kind of weird. Is it really nesting? It's certainly not hoon type nesting, for sure. idk if that'll lead to confusion down the line or not.

Reply:
You're right, that was one reason I decided to rewrite this section, but I forgot to change the title.